includes/admin/tools.php

```diff
@@ -1467,6 +1468,26 @@ function edd_tools_sysinfo_get() {
 	return $return;
 }
 
+/**
+ * Get TLS Info
+ *
+ * @since       2.9
+ * @return      string
+ */
+function edd_tools_tlsinfo() {
+
+	if( ! function_exists( 'curl_init' ) ) {
+		return 'TLS info unavaible while cURL is disabled';
+	}
+
+	$ch = curl_init( 'https://www.howsmyssl.com/a/check' );
```

Comment:
WordPress [recommends](https://github.com/WordPress/WordPress-Coding-Standards/blob/fab1ed525e057c19d723bf7c93feaf0a48512abe/WordPress/Sniffs/WP/AlternativeFunctionsSniff.php#L87) using wp_remote_get over cURL.

Reply:
For some reason when I use wp_remote_get it tells me "TLS Info:                 Err 0 The website calling howsmyssl.com's API has been making many calls and does not have a subscription. See https://subscriptions.howsmyssl.com/signup for how to get one."

But when I use cURL it's fine. Any idea why this might be?

Reply:
It appears GiveWP went through a similar discussion here: 
https://github.com/impress-org/givewp/issues/2998

Reply:
Oh that's great, thank you.

Reply:
So it looks like GiveWP's solution was to use file_get_contents.